wagtailio/project_styleguide/templates/patterns/components/skip-link/skip-link.html

```diff
@@ -0,0 +1 @@
+<a href="#main-content" class="skip-link">Skip to main content</a>
```

Comment:
Question: As this is aimed at screen readers, do we want to provide anything further for them? like aria label describing what will happen.

Reply:
I would have thought that text within the anchor link would be enough for them to determine what's going to happen. I can't [see any mention](https://webaim.org/techniques/skipnav/) of using an aria label so think this should suffice.